src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -719,4 +719,19 @@
   <data name="Profile_BellStyleVisual.Content" xml:space="preserve">
     <value>Visual (Flash Taskbar)</value>
   </data>
+  <data name="Profile_Acrylic.Text" xml:space="preserve">
+    <value>Acrylic</value>
+  </data>
+  <data name="Profile_Background.Text" xml:space="preserve">
+    <value>Background image</value>
+  </data>
+  <data name="Profile_Cursor.Text" xml:space="preserve">
+    <value>Cursor</value>
+  </data>
+  <data name="Profile_Text.Text" xml:space="preserve">
+    <value>Text</value>
+  </data>
+  <data name="Profile_Window.Text" xml:space="preserve">
+    <value>Window</value>
+  </data>

```

Comment:
Can you change the names of these to like `Profile_AcrylicHeader`? Or just differentiate them from settings somehow.